bundles/org.openhab.binding.zwavejs/src/main/java/org/openhab/binding/zwavejs/internal/handler/ZwaveJSNodeHandler.java

```diff
@@ -102,18 +106,26 @@ private static class ColorCapability {
         private boolean supportsWarmWhite;
         private boolean supportsColdWhite;
         private HSBType cachedColor;
+        private int cachedWarmWhite;
+        private int cachedColdWhite;
+        public @Nullable String colorTemperatureChannel;
 
         public ColorCapability() {
```

Comment:
Better to extract this class to a dedicated file

Reply:
Sorry. What are you referring to? And what do you mean by dedicated file?

Reply:
Oh. I understand. One class one file.

Reply:
Yes, exactly ;-)